### ISSUE TITLE
Support Chunked File Upload (larger file sizes than php limit)

### ISSUE THREAD

**Issue by reox** [nextcloud/deck#4376]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
Following the issue #1159, if the upload limit in PHP is restricted, you will get an error message for larger files on upload.
However, Nextcloud itself supports the upload of larger files flawlessly.

**Describe the solution you'd like**
Deck should use the same method of uploading files as nextcloud does, thus eliminating the file size limits of PHP.

**Describe alternatives you've considered**
Files can already be uploaded to the Deck folder and shared via the share from nextcloud button.

**Additional context**
-

**Comment by juliushaertl** [comment]
Makes sense, this could make use of https://github.com/nextcloud/nextcloud-upload
